recipes/qt5/meta.yaml

```diff
@@ -41,12 +52,12 @@ requirements:
     - curl  # [win]
     - flex  # [linux]
     - fontconfig 2.11.*  # [linux]
-    - freetype  2.5* # [linux]
+    - freetype  2.5*  # [linux]
     - gst-plugins-base  # [linux]
-    - icu 56.*
```

Comment:
I was aware of that, I was just trying to use packages from the default channels to be able to upload those packages to our official repos.